idunn/utils/categories.yml

```diff
@@ -1,144 +1,584 @@
-            - "collèges privés"
-            - "lycées d'enseignement général et technologique privés"
-            - "enseignement supérieur public"
-            - "enseignement supérieur privé"
-        raw_filters:
-            - "school,*"
-            - "college,*"
-        regex: "^ecole|^college|^universit|educat"
-    bar:
-        pj_what: "bars"
+          - "supermarchés, hypermarchés"
+
+    bank:
+        regex: "^(bank|banque|credit|caisse|atm$)"
+        raw_filters:
+          - "bank,*"
+        pj_what: "banque"
         pj_filters:
-            - "cafés, bars"
+          - "banques"
+
+    fast_food:
         raw_filters:
-            - "bar,*"
-            - "cafe,*"
+          - "fast_food,*"
+        pj_what: "fast food"
+
+    bar:
         regex: "^(cafes?|bars?|pubs?)( |$)"
-    museum:
-        pj_what: "musée"
+        raw_filters:
+          - "bar,*"
+          - "cafe,*"
+          - "pub,*"
+          - "biergarten,*"
+        pj_what: "bar"
         pj_filters:
-            - "musées"
+          - "cafés, bars"
+
+    hotel:
+        regex: "hotel"
         raw_filters:
-            - "museum,*"
-        regex: "^(museum|musee)"
-    health:
-        pj_what: "hopital"
+          - "lodging,*"
+        pj_what: "hôtels"
         pj_filters:
-            - "Hôpital"
+          - "hôtels"
+
+    historic:
         raw_filters:
-            - "clinic,*"
-            - "hospital,*"
-        regex: "health|sante|hopital|hospital|clinic|clinique"
-    service:
-        pj_what: "poste"
+          - "memorial,*"
+          - "monument,*"
+          - "statue,*"
+          - "castle,*"
+          - "citywalls,*"
+          - "fort,*"
+          - "battlefield,*"
+          - "castle_walls,*"
+          - "earthworks,*"
+          - "moat,*"
+          - "church,*"
+          - "tumulus,*"
+          - "stone_circle,*"
+          - "menhir,*"
+          - "standing_stone,*"
+          - "bridge,*"
+          - "milestone,*"
+          - "wall,*"
+          - "well,*"
+          - "folly,*"
+          - "ruins,*"
+
+    post_office:
+        raw_filters:
+          - "post_office,*"
+        pj_what: "bureau de poste"
+
+    fuel:
+        raw_filters:
+          - "fuel,*"
+        pj_what: "station essence"
+
+    community_centre:
+        raw_filters:
+          - "*,community_centre"
+
+    shop_convenience:
+        raw_filters:
+          - "*,convenience"
+          - "*,deli"
+        pj_what: "épicerie"
+
+    shop_car:
+        raw_filters:
+          - "*,car_repair"
+          - "*,car"
+        pj_what: "concessionnaire"
+
+    kindergarten:
+        raw_filters:
+          - "*,kindergarten"
+        pj_what: "crèche"
+
+    camp_site:
+        raw_filters:
+          - "camp_site,*"
+          - "caravan_site,*"
+        pj_what: "camping"
+
+    station:
+        regex: "^gare( |$)"
+        raw_filters:
+          - "railway,station"
+          - "railway,halt"
+        pj_what: "gare"
```

Comment:
Sorry for the oversight, I think that this `pj_what` should be removed here too. It was not present before.  
Maybe false positives are returned by PJ for such a query.

Apart from that, everything looks good to me :+1:

Reply:
That's fixed, sorry about this, my approach to this PR was absolutely messy and counter-productive ... I'm sorry about the time it must have taken to review and quite relieved it comes to an end :|